php/.tcr/toolchain/phpunit.yml

```diff
@@ -0,0 +1,20 @@
+build:
+  - os: [darwin, linux]
+    arch: ["386", amd64, arm64]
+    command: ./vendor/bin/phpunit
+    arguments: [--list-tests]
+  - os: [windows]
+    arch: ["386", amd64, arm64]
+    command: .\vendor\bin\phpunit
+    arguments: [--list-tests]
```

Comment:
From I could see, running `phpunit --list-tests` allows to detect syntax errors in the test files (which is good), but it does not seem to spot syntax error in source files. Do you know of any other php tool that could run syntax check on both src and test files?

Reply:
I've just tried an alternative tool to run the syntax check that seems to do the job: php-parallel-lint
When running the command `parallel-lint --exclude vendor .` I get RC=0 if all src and test files are syntaxically correct, and RC=1 when any of these is not